temporal-sdk/src/main/java/io/temporal/internal/sync/SyncWorkflowWorker.java

```diff
@@ -99,6 +96,22 @@ public SyncWorkflowWorker(
     workflowWorker =
         new WorkflowWorker(
             service, namespace, taskQueue, singleWorkerOptions, taskHandler, stickyTaskQueueName);
+
+    // it's just to support Worker#replayWorkflowExecution functionality
+    // This handler has to be non-sticky to don't evict actual executions from the cache
```

Comment:
```suggestion
    // Exists to support Worker#replayWorkflowExecution functionality.
    // This handler has to be non-sticky to avoid evicting actual executions from the cache
```